Palindrome.java

```diff
@@ -0,0 +1,17 @@
+class PalindromeExample{  
+ public static void main(String args[]){  
+  int r,sum=0,temp;    
+  int n=454;//It is the number variable to be checked for palindrome  
+  
+  temp=n;    
+  while(n>0){    
+   r=n%10;  //getting remainder  
+   sum=(sum*10)+r;    
+   n=n/10;    
+  }    
```

Comment:
I'm not sure about this, and didn't got the time to test if it actually works, but I would suggest to verify the palindrome by converting it to String, and treat it as a string.